pkg/rpc/client/wsclient.go

```diff
@@ -21,17 +21,17 @@ import (
 
 // WSClient is a websocket-enabled RPC client that can be used with appropriate
 // servers. It's supposed to be faster than Client because it has persistent
-// connection to the server and at the same time is exposes some functionality
+// connection to the server and at the same time shows some functionality
```

Comment:
I'd do s/is/it/ as an obvious typo correction, but it really exposes things as in making them available, not demonstrating.